### PR TITLE
Optimizations: combine cross-variant scripts, tightened tests

### PR DESCRIPTION
**Pull request by bryanlatten**
- Using latest parent (6.2)
- Additional opcache configuration via env variables
- Tuning/tweaks now use same commands across all variants
- Base tests now run in all variants
- Relying on single (non-7.0) symlink for start command
- Upgraded NewRelic on beta version (only), which is now officially supporting PHP 7.1

### CODE REVIEW

**Comment by ko-be** [comment]
👍  after tests pass!

File: Dockerfile-edge [hunk 1]
```diff
@@ -15,7 +15,10 @@ ENV CONF_PHPFPM=/etc/php/7.1/fpm/php-fpm.conf \
     PHP_FPM_MEMORY_LIMIT=256M \
     PHP_FPM_MAX_EXECUTION_TIME=60 \
     PHP_FPM_UPLOAD_MAX_FILESIZE=1M \
-    NEWRELIC_VERSION=6.7.0.174 \
+    PHP_OPCACHE_MEMORY_CONSUMPTION=128 \
+    PHP_OPCACHE_INTERNED_STRINGS_BUFFER=16 \
+    PHP_OPCACHE_MAX_WASTED_PERCENTAGE=5 \
+    NEWRELIC_VERSION=6.8.0.177 \
```
Comment (bryanlatten) [inline Dockerfile-edge:26]: new version supports 7.1

File: Dockerfile-alpine [hunk 2]
```diff
@@ -96,61 +99,14 @@ RUN curl -L https://download.newrelic.com/php_agent/archive/${NEWRELIC_VERSION}/
...
-    sed -i "s/;*request_terminate_timeout = .*/request_terminate_timeout = \${PHP_FPM_MAX_EXECUTION_TIME}/" $CONF_FPMPOOL && \
-    # Required for php-fpm to place .sock file into, fails otherwise \
-    mkdir -p /var/run/php/ /var/log/php7/newrelic && \
-    chown -R $NOT_ROOT_USER:$NOT_ROOT_USER /var/run/php /var/log/newrelic
-
 # Overlay the root filesystem from this repo
 COPY ./container/root /
 
-# Make additional hacks to migrate files from Ubuntu to Alpine folder structure
+# - Make additional hacks to migrate files from Ubuntu to Alpine folder structure \
```
Comment (mlebrun) [inline Dockerfile-alpine]: does this commented line need a slash at the end?

File: Dockerfile-edge [hunk 3]
```diff
@@ -95,68 +98,29 @@ RUN apt-get update -q && \
     # Install new PHP7-stable version of Yaml \
     pecl install yaml-2.0.0 && \
     echo "extension=yaml.so" > $CONF_PHPMODS/yaml.ini && \
-    # Install new PHP7-stable version of Redis \
+    # Install new PHP7-stable version of Redis
     pecl install redis-3.0.0 && \
```
Comment (mlebrun) [inline Dockerfile-edge:101]: what's Redis being used for? just curious

File: Dockerfile-alpine (hunk 2 above)
Comment (bryanlatten) [reply Dockerfile-alpine]: in fact, none of the comments do

File: Dockerfile-edge (hunk 3 above)
Comment (bryanlatten) [reply Dockerfile-edge:101]: its not being used by default, but it is available for teams to use, which do exist today

**Comment by mlebrun** [comment]
👍

**Comment by jimdelois** [comment]
👍

**Comment by mannytoledo** [comment]
👍

**Comment by bossjones** [comment]
👍

**Pull request merged**